src/ValidatorErrorIterator.php

```diff
@@ -44,7 +44,7 @@ public function withSourcePrefix(string $prefix): self
         $prefix = rtrim($prefix, '/');
```

Comment:
As per my comment in the issue, change this to:

```php
$prefix = trim($prefix, '/');
```

As we want to always ensure it starts with a slash too.